src/app/model/event/event.py

```diff
@@ -34,12 +35,21 @@ def __init__(self, d=None):
     def get_score(self):
         block: Block = Munch.fromDict(self.blocks[0])
         elements: [Element] = block.elements[0].elements
-        score_text = [elem for elem in elements if (hasattr(elem, "text") and "/5" in elem.text)][0].text
-        score = re.split(' ', score_text, 1)[1][0]
-        return int(0 if score == 'X' else score[0])
+        score_text = self._search_elements_for_expr(elements, "#waffle[0-9]+ [0-5]/[0-5]")
+        if score_text is not None:
+            score = re.split(' ', score_text, 1)[1][0]
+            return int(0 if score == 'X' else score[0])
 
     def get_streak(self):
         block: Block = Munch.fromDict(self.blocks[0])
         elements: [Element] = block.elements[0].elements
-        streak_element: Element = [elem for elem in elements if (hasattr(elem, "text") and "streak" in elem.text)][0]
-        return int(streak_element.text.split(" ")[2].strip("\n"))
+        streak_text = self._search_elements_for_expr(elements, "streak: [0-9]*\n")
```

Comment:
Does this still need `self.`?

Reply:
Apparently so:

```

self = <src.app.model.event.event.Event object at 0x0000018FA4059810>

    def get_score(self):
        block: Block = Munch.fromDict(self.blocks[0])
        elements: [Element] = block.elements[0].elements
>       score_text = _search_elements_for_expr(elements, "#waffle[0-9]+ [0-5]/[0-5]")
E       NameError: name '_search_elements_for_expr' is not defined

src\app\model\event\event.py:38: NameError
============================================================================================================================================= short test summary info ============================================================================================================================================= 
FAILED src\app\model\event\event_spec.py::EventSpec::should_extract_when_waffle_is_only_text_present - NameError: name '_search_elements_for_expr' is not defined
FAILED src\app\model\event\event_spec.py::EventSpec::should_extract_when_waffle_is_before_short_text - NameError: name '_search_elements_for_expr' is not defined
FAILED src\app\model\event\event_spec.py::EventSpec::should_extract_when_waffle_is_after_short_text - NameError: name '_search_elements_for_expr' is not defined
FAILED src\app\model\event\event_spec.py::EventSpec::should_extract_when_waffle_is_before_long_text - NameError: name '_search_elements_for_expr' is not defined
FAILED src\app\model\event\event_spec.py::EventSpec::should_extract_when_waffle_is_after_long_text - NameError: name '_search_elements_for_expr' is not defined
FAILED src\app\model\event\event_spec.py::EventSpec::should_extract_when_waffle_is_after_false_flags - NameError: name '_search_elements_for_expr' is not defined
FAILED src\app\model\event\event_spec.py::EventSpec::should_not_extract_from_invalid_post - NameError: name '_search_elements_for_expr' is not defined
========================================================================================================================================== 7 failed, 14 passed in 0.86s =========================================================================================================================================== 
make: *** [Makefile:23: test] Error 1

```

Reply:
There's documentation to explain why this is the case: https://peps.python.org/pep-0008/#method-names-and-instance-variables